api/base/caller.go

```diff
@@ -145,11 +145,15 @@ func NewFacadeCaller(caller APICaller, facadeName string, options ...Option) Fac
 // NewFacadeCallerForVersion wraps an APICaller for a given facade
 // name and version.
 func NewFacadeCallerForVersion(caller APICaller, facadeName string, version int, options ...Option) FacadeCaller {
+	// Derive the context from the API caller context if it's available. The
+	// default will be a noop tracer if none is found.
+	tracer, _ := coretrace.TracerFromContext(caller.Context())
```

Comment:
Is the bool return value for TracerFromContext ever used?

Reply:
Yeah, internally in the txn runner (PR https://github.com/juju/juju/pull/16402), we need to know if it's actually enabled from just the context side.